linetools/spectra/lsf.py

```diff
@@ -230,17 +231,36 @@ def interpolate_to_wv0(self,wv0):
         #transform to wavelength in float() form assuming Angstroms
         col_names = self._data.keys()
         col_waves = [float(name.split('A')[0]) for name in col_names[1:]]
-                    
+
         lsf_vals = []
         for row in self._data:
             aux_val = []
             for i in range(1,len(row)):
                 aux_val += [row[i]]
-            f = interp1d(col_waves,aux_val,bounds_error=True,kind='linear') #we do not want to extrapolate
-            lsf_vals += [f(wv0.value)]
+            #we don't want to extrapolate wildly, but allow small extrapolations for wv0
+            if (wv0.value >= col_waves[0]) & (wv0.value <= col_waves[-1]):
+                f = interp1d(col_waves,aux_val,bounds_error=True,kind='linear') #no need to extrapolate
+                lsf_vals += [f(wv0.value)]
+
+            elif (wv0.value < col_waves[0]) & \
+                    ((col_waves[0]-wv0.value) < np.abs(col_waves[1]-col_waves[0])):
+                f = interp1d(col_waves, aux_val, bounds_error=False,
+                             fill_value=aux_val[0], kind='linear')  #assign shortest wv LSF definition
+                lsf_vals += [f(wv0.value)]
+
+            elif (wv0.value > col_waves[-1]) & ((wv0.value - col_waves[-1]) < np.abs(col_waves[-1] - col_waves[-2])):
```

Comment:
Uh-oh, just testing on CASBaH data that extend to 1800 Angstroms.  The LSF definition file only goes to 1750.  I'm extending the range on long-wavelength end.  In general, the NUV will take over beyond 1800 \AA, but we have lines right out to 1800 covered by G160M.  I'll leave the shorter extrapolation range on the short-wavelength end (~1150 \AA).